pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -76,9 +76,9 @@ func (s *spdxParser) getTopLevelPackage() error {
 	splitImage := strings.Split(s.spdxDoc.DocumentName, "/")
 	var purl string
 	if len(splitImage) == 3 {
```

Comment:
can remove this check and `split` above as not needed.

Reply:
+1

Reply:
apparently email replies to the entire issue, bringing the comments back to this thread 

>@lumjjb: We still need the check since it may not be an oci image
>@pxp928: Does it need to be type oci? Seems very error prone again splitting strings.

Reply:
well i think its a guac heuristic for OCI images, so maybe we can make it `guac/heuristic/oci/path`?